src/test/java/uk/gov/hmcts/reform/civil/bpmn/DefendantResponseCuiTest.java

```diff
@@ -106,6 +106,8 @@ void shouldNotNotifyOrGeneratePdf_whenDefendantResponseBilingual() {
 
```

Comment:
need to change the name of the test to just not notify applicant

Reply:
done